spec/support/shared_examples/participant_actions_resume_support.rb

```diff
@@ -19,3 +19,79 @@
     expect { described_class.call(params: given_params) }.to raise_error(ActiveRecord::RecordInvalid)
   end
 end
+
+RSpec.shared_examples "JSON Participant Resume endpoint" do |serialiser_type|
+  let(:parsed_response) { JSON.parse(response.body) }
+
+  it "changes the training status of a participant to active" do
+    put url, params: params
+
+    expect(response).to be_successful
+
+    expect(parsed_response.dig("data", "type")).to eq(serialiser_type)
+  end
+
+  it "returns an error when the participant is already active" do
+    2.times { put url, params: params }
+
+    expect(response).not_to be_successful
+  end
+
+  it "returns an error when the participant is withdrawn" do
```

Comment:
Why are we testing the withdrawal endpoint in here as well? isn't this just supposed to be about resume?